Main.as

```diff
@@ -1,13 +1,19 @@
 CHmsCamera@ g_currentCamera = null;
 
 mat4 g_projection = mat4::Identity();
+mat4 g_rotation = mat4::Identity();
 vec3 g_position = vec3();
+vec3 g_direction = vec3();
 
 vec2 g_displayPos;
 vec2 g_displaySize;
 
+ActiveCam g_activeCam = ActiveCam::None;
```

Comment:
Maybe `g_activeCameraMode` would be a better name for this?

Reply:
I changed it to `g_activeCameraType` ("type" makes more sense to me compared to "mode")